src/seedu/addressbook/Main.java

```diff
@@ -81,7 +81,8 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
+            new Parser();
```

Comment:
If all of your methods in the `Parser` class are already `static`, do we still need to initialize an instance of `Parser`? Think about how we normally use the `Math` class in Java API.